chartfx-math/src/main/java/de/gsi/math/MathBaseGenBase.java

```diff
@@ -0,0 +1,428 @@
+     */
+    public static final double QE = 1.602176462e-19;
+    /**
+     * uncertainty of the elementary charge
+     */
+    public static final double QE_UNCERTAINTY = 0.000000063e-19;
+    /**
+     * Planck's constant [J s]
+     */
+    public static final double H = 6.62606876e-34;
+    /**
+     * h-bar (h over 2 pi) [J s]
+     */
+    public static final double H_BAR = 1.054571596e-34;
+    /**
+     * uncertainty of h-bar
+     */
+    public static final double H_BAR_UNCERTAINTY = 0.000000082e-34;
+    /**
+     * h-bar in [erg s]
+     */
+    public static final double H_BAR_CGS = 1.0e7 * H_BAR;
+    /**
+     * hc (h * c) [J m]
+     */
+    public static final double HC = H * C;
+    /**
+    * Boltzmann's constant [J K^-1]
+     */
+    public static final double K = 1.3806503e-23;
+    /**
+     * uncertainty of Boltzmann's constant
+     */
+    public static final double K_UNCERTAINTY = 0.0000024e-23;
+    /**
+     * Boltzmann's constant in  [erg K^-1]
+     */
+    public static final double K_CGS = 1.0e7 * K;
+    /**
+     * Stefan-Boltzmann constant [W m^-2 K^-4]
+     */
+    public static final double SIGMA = 5.6704e-8;
+    /**
+     * Uncertainty of Stefan-Boltzmann constant
+     */
+    public static final double SIGMA_UNCERTAINTY = 0.000040e-8;
+    /**
+     * Avogadro constant (Avogadro's Number) [mol^-1]
+     */
+    public static final double N_A = 6.02214199e+23;
+    /**
+     * Uncertainty of the Avogadro constant
+     */
+    public static final double N_A_UNCERTAINTY = 0.00000047e+23;
+    /**
+     * universal gas constant (Na * K) [J K^-1 mol^-1]
+     * http://scienceworld.wolfram.com/physics/UniversalGasConstant.html
+     */
+    public static final double R = K * N_A;
+    /**
+     * Uncertainty of the Dry Air Gas Constant
+     */
+    public static final double H_UNCERTAINTY = 0.00000052e-34;
+    /**
+     * Dry Air Gas Constant (R / MWair) [erg s]
+     * http://atmos.nmsu.edu/education_and_outreach/encyclopedia/gas_constant.htm
+     */
+    public static final double H_CGS = 1.0e7 * H;
+    /**
+     * h_cgs * c_cgs [erg cm]
+     */
+    public static final double HC_CGS = H_CGS * C_CGS;
+    /**
+     * Molecular weight of dry air [kg kmol^-1 (or gm mol^-1)]
+     *  1976 US Standard Atmosphere,
+     *  also see http://atmos.nmsu.edu/jsdap/encyclopediawork.html
+     */
+    public static final double MW_AIR = 28.9644;
+    /**
+     *  [J kg^-1 K^-1]
+     */
+    public static final double RG_AIR = (1000.0 * R) / MW_AIR;
+    /**
+     *
+     */
+    public static final double R_UNCERTAINTY = R * ((K_UNCERTAINTY / K) + (N_A_UNCERTAINTY / N_A));
+
+    public static double log(double x) {
+        return java.lang.Math.log(x);
+    }
+
+    public static double log10(double x) {
+        return java.lang.Math.log10(x);
+    }
+
+    public static double aCosH(double arg0) {
+        if (arg0 == 0.0)
+            return 0.0;
+        double ax = java.lang.Math.abs(arg0);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.abs' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)